Yafc/Windows/NeverEnoughItemsPanel.cs

```diff
@@ -91,18 +92,20 @@ private enum EntryStatus {
 
             this.current = current;
             currentFlow = current.ApproximateFlow(atCurrentMilestones);
-            productions.Clear();
```

Comment:
I think that naming the local variable the same as the class field can lead to confusing the two by someone who reads the code next. What do you guys think?

Reply:
It's already done with `this.current = current` :) I don't find this confusing, but I understand someone might, would you prefer to name it `newCurrent`/`newProductions`/`newUsages`?

Reply:
Either way is fine for me

Reply:
> would you prefer to name it

I think something like `refreshedProductions` and `refreshedUsages` would convey that these variables are meant as a storage for a new list. The difference in their name compared to the corresponding fields would help to distinguish them.

Reply:
Change the name as you suggest.